strip_url_params.py : BUG in function strip_url_params3(...)

The file ```strip_url_params.py``` is buggy. Because one times the tests run successful and a other times it fails. I comment out the test for this file.  

***important:*** The bug is in the function ```strip_url_params3``` of the file above.  

@SaadBenn Can you look in it? The project is from you.

@danghai The error message of the failed test looks so
> line 330, in test_strip_url_params3
    self.assertEqual(strip_url_params3("www.saadbenn.com?a=1&b=2&a=2"), "www.saadbenn.com?a=1&b=2")
AssertionError: 'www.saadbenn.com?b=2&a=1' != 'www.saadbenn.com?a=1&b=2'
> - www.saadbenn.com?b=2&a=1
> ?                     ----
> + www.saadbenn.com?a=1&b=2
> ?                  ++++

--- a/algorithms/strings/strip_url_params.py
+++ b/algorithms/strings/strip_url_params.py
@@ -10,6 +10,8 @@ from collections import defaultdict
 import urllib
 import urllib.parse
 
+from collections import OrderedDict
+
 # Here is a very non-pythonic grotesque solution
 def strip_url_params1(url, params_to_strip=None):
     
@@ -37,7 +39,7 @@ def strip_url_params1(url, params_to_strip=None):
                     string = ''
                 else:
                     string += char
-            dict = defaultdict(int)
+            dict = OrderedDict()
             # logic for checking whether we should add the string to our result
             for i in key_value_string:
                 _token = i.split('=')

--- a/tests/test_strings.py
+++ b/tests/test_strings.py
@@ -339,23 +339,23 @@ class TestRomanToInt(unittest.TestCase):
         self.assertEqual(3999, roman_to_int("MMMCMXCIX"))
 
 
-# class TestStripUrlParams(unittest.TestCase):
-#     """[summary]
-#     Test for the file strip_urls_params.py
+class TestStripUrlParams(unittest.TestCase):
+    """[summary]
+    Test for the file strip_urls_params.py
 
-#     Arguments:
-#         unittest {[type]} -- [description]
-#     """
+    Arguments:
+        unittest {[type]} -- [description]
+    """
 
-#     def test_strip_url_params1(self):
-#         self.assertEqual(strip_url_params1("www.saadbenn.com?a=1&b=2&a=2"), "www.saadbenn.com?a=1&b=2")
-#         self.assertEqual(strip_url_params1("www.saadbenn.com?a=1&b=2", ['b']), "www.saadbenn.com?a=1")
-#     def test_strip_url_params2(self):
-#         self.assertEqual(strip_url_params2("www.saadbenn.com?a=1&b=2&a=2"), "www.saadbenn.com?a=1&b=2")
-#         self.assertEqual(strip_url_params2("www.saadbenn.com?a=1&b=2", ['b']), "www.saadbenn.com?a=1")
-#     def test_strip_url_params3(self):
-#         self.assertEqual(strip_url_params3("www.saadbenn.com?a=1&b=2&a=2"), "www.saadbenn.com?a=1&b=2")
-#         self.assertEqual(strip_url_params3("www.saadbenn.com?a=1&b=2", ['b']), "www.saadbenn.com?a=1")
+    def test_strip_url_params1(self):
+        self.assertEqual(strip_url_params1("www.saadbenn.com?a=1&b=2&a=2"), "www.saadbenn.com?a=1&b=2")
+        self.assertEqual(strip_url_params1("www.saadbenn.com?a=1&b=2", ['b']), "www.saadbenn.com?a=1")
+    def test_strip_url_params2(self):
+        self.assertEqual(strip_url_params2("www.saadbenn.com?a=1&b=2&a=2"), "www.saadbenn.com?a=1&b=2")
+        self.assertEqual(strip_url_params2("www.saadbenn.com?a=1&b=2", ['b']), "www.saadbenn.com?a=1")
+    def test_strip_url_params3(self):
+        self.assertEqual(strip_url_params3("www.saadbenn.com?a=1&b=2&a=2"), "www.saadbenn.com?a=1&b=2")
+        self.assertEqual(strip_url_params3("www.saadbenn.com?a=1&b=2", ['b']), "www.saadbenn.com?a=1")
 
 
 class TestValidateCoordinates(unittest.TestCase):

